lib/invoice_item_repository.rb

```diff
@@ -1,4 +1,4 @@
-require_relative 'file_loader'
+require_relative 'load_file'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.